staging/cse/windows/kubernetesfunc.ps1

```diff
@@ -178,21 +178,30 @@ function Check-APIServerConnectivity {
     $retryCount=0
 
     do {
-        try {
-            $tcpClient=New-Object Net.Sockets.TcpClient
-            Write-Log "Retry $retryCount : Trying to connect to API server $MasterIP"
-            $tcpClient.ConnectAsync($MasterIP, 443).wait($ConnectTimeout*1000)
-            if ($tcpClient.Connected) {
+        $retryString="${retryCount}/${MaxRetryCount}"
+        try
+        {
+            $tcpClient = New-Object Net.Sockets.TcpClient
+            $tcpClient.SendTimeout = $ConnectTimeout*1000

```

Comment:
is ConnectTimeout in MS?

Reply:
It's in Seconds. 

![image](https://github.com/user-attachments/assets/b46d1b8c-ce01-4cf5-b861-2d0c2d16638d)